### PR TITLE
feat(preset): add eslint preset

### PR DESCRIPTION
**Pull request by fxmanu**

### CODE REVIEW

**Comment by stevemao** [comment]
Hey @fxmanu , I'm actually going to remove presets since nobody is using it. https://github.com/conventional-changelog/conventional-recommended-bump/issues/16
What do you think?

**Comment by fxmanu** [comment]
@stevemao I think the preset feature might be useful. But if really nobody is using it, removing it seems legit.

What do you mean by "use config" in #16 ?

**Comment by stevemao** [comment]
Sorry, just mean that loading a config file should be supported to replace preset.

**Comment by fxmanu** [comment]
That would be even better!

File: presets/eslint.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        features += 1;
+        if (level === 2) {
+          level = 1;
+        }
+      }
+    });
+
+    return {
+      level: level,
+      reason: 'There are ' + breakings + ' BREAKING CHANGES and ' + features + ' features'
```
Comment (hutson) [inline presets/eslint.js:22]: I wonder if this text could be shared between presets?

**Comment by hutson** [comment]
@stevemao @fxmanu using a configuration file to drive the choice of preset is being discussed in conventional-changelog/conventional-changelog/issues/195

In the meantime, @stevemao what do you think of having @fxmanu re-file this enhancement against the `conventional-recommended-bump` package in the `conventional-changelog` monorepo?
 (https://github.com/conventional-changelog/conventional-changelog)

I'd like to see users have access to another awesome preset for their projects!

**Comment by hutson** [comment]
I've been given permissions to close out issues and pull requests for the purpose of moving discussions over to the `conventional-changelog` monorepo.

In the meantime, as I mentioned in my previous comment, while the community is discussing how to move forward with support for per-project configuration files, and the proper behavior for the `preset` option, I'd like to encourage you, @fxmanu, to re-submit this enhancement to `conventional-recommended-bump` within the `conventional-changelog` monorepo if you would find value in these changes in near future.

**Pull request closed without merging**